### DISCUSSION TITLE
Questions about differences in test results

### DISCUSSION THREAD

**Discussion by 20210070Kimdonghan** [ANLAB-KAIST/KENSv3#221] [Project Q&A 2. Transport Layer (TCP)]
Hello,

My team is testing the project 3, and I found that the test results did not match on my computer. It has been confirmed that my computer gives the wrong answer, while my colleague's computer outputs the correct answer.

I'd like to ask you a few questions regarding this:

Is there anything I need to check if there is anything missing about the configuration settings or dependencies needed to run the test?
I understand that the hardware or software configuration is different between my computer and my colleague's, could this lead to a different outcome? My computer is samsung and my colleague's computer is max.
Is there anything I need to check if there is a difference in the data or input values used to run the test?
Is there anything I need to check for possible bugs in the test script or code?
I would appreciate it if you could help us to resolve these differences and achieve consistent results through these questions.

Thank you.

**Comment by taehokimmm** [comment]
I am on the same team with him, and we think we sorted it out. Apparently the behaviour of the test cases between Linux and Mac computers were kinda different even if the `RANDOM_SEED` was identical.

**Reply by Pusnow** [reply]
Right, it is because their random generators are different.
